rdata/io/xdr.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from typing import (
+    Any,
+    BinaryIO,
+)
+
+import numpy as np
+import numpy.typing as npt
+
+from rdata.parser._parser import (
+    AltRepConstructorMap,
+    DEFAULT_ALTREP_MAP,
+    Parser,
+    RData,
+)
```

Comment:
Related to restructuring, this would probably need changes to avoid importing from `_parser`.

Reply:
I agree, I'll move it. I had added the writers within the same files (XDR writer and parser both in `rdata/io/xdr.py`), but those would probably be better organized other way too.